bitnami/harbor/templates/clair/clair-dpl.yaml

```diff
@@ -15,7 +15,7 @@ spec:
       labels: {{- include "harbor.labels" . | nindent 8 }}
         app.kubernetes.io/component: clair
       annotations:
-        checksum/configmap: {{ include (print $.Template.BasePath "/clair/clair-cm.yaml") . | sha256sum }}
```

Comment:
Why are you doing this? This file has no password inside it is correct to be a configmap.

Reply:
Answered in https://github.com/bitnami/charts/pull/1941/files#r380523144

Reply:
For that case I think it should be enough with the `redis.password` parameter

Reply:
Please, take into account that I am using the externalRedis feature, so I have to set `redis.enabled: false` to avoid installing stable/redis that I won't use.